dynamic_programming/longest_palindromic_subsequence.py

```diff
@@ -0,0 +1,56 @@
+"""
+Longest Palindromic Subsequence Problem:
+Find the longest sequence of characters from a given input string that
+is the same backwards and forwards
+https://en.wikipedia.org/wiki/Longest_common_subsequence_problem
+"""
+
+
+def LongestPalindromicSubsequence(string, n):
```

Comment:
```suggestion
def longest_palindromic_subsequence(s: str, n: int) -> str:
```
Python function names are in snake_case as discussed in CONTRIBUTING.md.

Python type hints as discussed in CONTRIBUTING.md.

`string` is the name of a Python standard library module so should not be used as a variable name.

Can we come up with a more self-documenting variable name than `n`?